### PR TITLE
Add tutorial

### LINKED ISSUE #25: Create tutorial/example how to use openshift2nulecule

### PR DESCRIPTION
**Pull request by kadel**
fixes #25

### CODE REVIEW

**Comment by kadel** [comment]
@surajssd what are your thoughts on this? :wink:

File: docs/example.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+    - `cd adb-atomic-developer-bundle/components/centos/centos-openshift-setup`
+    - `vagrant up`
+    - `vagrant ssh`
+
+ 1. Add Docker Registry on your local machine as insecure
+     
+    - `sudo vi /etc/sysconfig/docker`
+    - Uncommend INSECURE_REGISTRY line and add your local Docker Registry (`'--insecure-registry=10.1.2.1:5000'`)
+    - `sudo systemctl restart docker`
```
Comment (surajssd) [inline docs/example.md]: Typo: s/Uncommend/Uncomment and adding INSECURE_REGISTRY as `INSECURE_REGISTRY` would make it readable to what is to be edited!

File: docs/example.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    `git clone https://github.com/projectatomic/adb-atomic-developer-bundle.git`
+
+ 1. Run OpenShift instance in ADB
+
+    - `cd adb-atomic-developer-bundle/components/centos/centos-openshift-setup`
+    - `vagrant up`
+    - `vagrant ssh`
+
+ 1. Add Docker Registry on your local machine as insecure
+
```
Comment (surajssd) [inline docs/example.md]: Its little unclear where to run this?

`If its to be run on my bare metal host:`
Please move this section up, put it before '*Start ADB and deploy sample OpenShift application*' and after '*Deploy Docker registry on your local machine*'

`else:`
Change the name **local machine** to **ADB box**.

Comment (surajssd) [inline docs/example.md]: If there is an issue of IP address getting assigned after the openshift box starts working then put the whole section of things to do on local machine after the openshift box setup has been done.

File: docs/example.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    - install openshift2nulecule
+
+       `sudo yum install openshift2nulecule`
+
+ 1. Get address of OpenShift internal Docker registry 
+     
+    ```
+    export OC_REGISTRY=$(sudo oc --config /var/lib/origin/openshift.local.config/master/admin.kubeconfig get svc docker-registry -o template --template="{{ .spec.clusterIP }}"):5000
+    ```
```
Comment (surajssd) [inline docs/example.md]: Typo in path to `kubeconfig`, correct path is `/var/lib/openshift/openshift.local.config/master/admin.kubeconfig`

File: docs/example.md (hunk 1 above)
Comment (kadel) [inline docs/example.md]: thx, will fix that

File: docs/example.md (hunk 3 above)
Comment (kadel) [inline docs/example.md]: not really a typo :-(
`/var/lib/origin` is correct path for ADB 1.7.0 it got changed in 1.7.1 to `/var/lib/origin`
But you are right, I have to fix that.

Did you test all this on  ADB 1.7.1?

File: docs/example.md (hunk 2 above)
Comment (kadel) [inline docs/example.md]: > Its little unclear where to run this?
> 
> If its to be run on my bare metal host:
> Please move this section up, put it before 'Start ADB and deploy sample OpenShift application' and after 'Deploy Docker registry on your local machine'
> 
> else:
> Change the name local machine to ADB box.

I have to rephrase that.
What I mean by this is: "Add address of Docker Registry that is running on you local machine (that was deployed in first step of this tutorial) as insecure registry inside ADB"

**Comment by kadel** [comment]
@surajssd  I updated commit to address issues you pointed out.

**Comment by kadel** [comment]
fixed

File: docs/example.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+ 1. Run OpenShift instance in ADB
+
+    - `cd adb-atomic-developer-bundle/components/centos/centos-openshift-setup`
+    - `vagrant up`
+    - `vagrant ssh`
+
+ 1. Add Docker Registry that runs on your local machine as insecure registry (in ADB box)
+     
+    - `sudo vi /etc/sysconfig/docker`
+    - Uncommend INSECURE_REGISTRY line and add your local Docker Registry (`'--insecure-registry=10.1.2.1:5000'`)
```
Comment (surajssd) [inline docs/example.md]: @kadel I found that this line is already uncommented, so why not rephrase this as:

Add your local machine's Docker registry (which was started in the first step) by appending `--insecure-registry 10.1.2.1:5000` to `INSECURE_REGISTRY`

Comment (kadel) [inline docs/example.md]: thx for pointing this out. That is another change between  ADB 1.7.0 and 1.7.1 :-(

**Comment by kadel** [comment]
another update ;-)

File: docs/example.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+## Run exported Nulecule application on Kubernetes
+ 1. Run Kubernetes instance in ADB
+   - `cd components/centos/centos-k8s-singlenode-setup`
+   - `vagrant up`
+   - `vagrant ssh`
+
+ 1. Add Docker Registry on you local machine as insecure
+     
+    - `sudo vi /etc/sysconfig/docker`
+    - Change line `# INSECURE_REGISTRY='--insecure-registry '` to `INSECURE_REGISTRY='--insecure-registry=10.1.2.1:5000'`
```
Comment (surajssd) [inline docs/example.md]: only one small change `INSECURE_REGISTRY='--insecure-registry=10.1.2.1:5000'` to `INSECURE_REGISTRY='--insecure-registry 10.1.2.1:5000'` only remove the `=`.

One query: Even if the k8s machine has no interface of IP `10.1.2.1` how is it able to talk to Host machine with that address?

Comment (kadel) [inline docs/example.md]: > only one small change INSECURE_REGISTRY='--insecure-registry=10.1.2.1:5000' to INSECURE_REGISTRY='--insecure-registry 10.1.2.1:5000' only remove the =.

done

> One query: Even if the k8s machine has no interface of IP 10.1.2.1 how is it able to talk to Host machine with that address?

Default route in ADB is set to host machine ip (because of NAT networking).
10.1.2.1 is also attached to one of the virtual interfaces on host machine so it is able to communicate with it
